### DISCUSSION TITLE
goodbyedpi anti-cheat problemi

### DISCUSSION THREAD

**Discussion by bthnsvt** [cagritaskn/GoodbyeDPI-Turkey#3] [Q&A] [answered]
call of duty blackops 6'nin anti-cheat programi windivert servisini hile olarak algiliyor. Bu  yuzden goodbyedpi acik iken bu oyuna giremiyorsunuz.

windivert servisini durdurursak aciliyor fakat bu sefer de bir sure sonra discorddan dusme yasaniyor. 

bu konu hakkinda bilgisi olan yardimci olabilirse sevinirim topluluk icin de yararli olacaktir

**Comment by cagritaskn** [comment, marked as answer]
Referenced to: https://github.com/cagritaskn/GoodbyeDPI-Turkey/discussions/2

**Comment by Lorsius** [comment]
service_remove.cmd bu isimli dosyayı çalıştırdım internetim gitti res atmamı istedi attım şimdi hiç bir şey kullanmadan her yere girebiliyorum ama aşırı pingim var yardımcı olabilir misiniz acaba

**Reply by cagritaskn** [reply]
Çalıştır'ı açıp services.msc yazın ve listeden GoodbyeDPI isimli hizmeti bulun. Eğer listede hâla bulunuyorsa başarılı şekilde kaldıramamışsınız demektir. Servise sağ tıklayıp başlangıç türünü devre dışı olarak ayarlayın ve aşağıdaki durdur butonuna basarak servisi durdurun.

Ya da cmd istemcisini açıp sırasıyla:

sc stop GoodbyeDPI
sc stop WinDivert
sc stop WinDivert1.4
sc stop Windivert2.2

yazarak hizmetleri durdurabilirsiniz.

GoodbyeDPI, sistem genelinde değişiklik yapma prensibiyle değil; geçici olarak çalıştırıldığı süre zarfında paket trafiğini düzenleme prensibiyle çalışır.
Dolayısıyla hizmet kurarak kullanıyorsanız hizmeti service_remove.cmd dosyasını yönetici olarak çalıştırdıktan sonra, geçiçi olarak batch dosyası ile kullanıyorsanız pencereyi kapattıktan sonra işlevi tamamen duruyor.

Bilgisayarınızda bir VPN ya da proxy kullanıyorsanız sitelere girebilmenizin ve pinginizin yüksek olmasının sebebi bu olabilir.
